lib/linguist/generated.rb

```diff
@@ -577,5 +578,17 @@ def generated_perl_ppport_header?
     def generated_graphql_relay?
       !!name.match(/__generated__\//)
     end
+
+    # Internal: Is this a generated Game Maker Studio (2) metadata file?
+    #
+    # All Game Maker Studio 2 generated files will be JSON, .yy or .yyp, and have
+    # a part that looks like "modelName: GMname" on the 3rd line
+    #
+    # Return true or false
+    def generated_gamemakerstudio?
+      return false unless extname == '.yy' || extname == '.yyp'
+      return false unless lines.count > 5
+      return /\"modelName\"\:\s*\"GM/.match(lines[3])
```

Comment:
In the comment you mention the third line, but here you match on the fourth. Typo?

Reply:
Sorry, yeah, was already on it ^_^

Reply:
(can't run the tests locally so abusing Travis for it :x )